web/src/components/storage/SpacePolicyDialog.jsx

```diff
@@ -149,7 +149,8 @@ in the devices listed below. Choose how to do it.");
       title={_("Find space")}
       description={description}
       isOpen={isOpen}
-      variant="medium"
+      blockSize="medium"
```

Comment:
I would use "large" for the blockSize here. In my system only two lines of the table are visible which is not enough to make sense out of the table.

![not-enough](https://github.com/openSUSE/agama/assets/3638289/3cccc59c-d95e-4a4b-a759-9f7a915a0c4b)

Sometimes there are several disks involved at the installation. Or maybe just one disk but with many partitions (bear in mind the list of partitions is expanded automatically when clicking on "custom"). In all those cases, a lot of information is not visible at first sight which makes it harder to understand how to use the whole thing.

If you think "large" is too much, maybe we can use an approach similar to the one in the list of actions, deciding the size based on the number of disks and partitions.

Reply:
> If you think "large" is too much, maybe we can use an approach similar to the one in the list of actions, deciding the size based on the number of disks and partitions.

I prefer this option, but bearing in mind that it is not so straightforward and I expect more changes in this dialog in the short/mid term, let's go for `blockSize="large"` as you suggested.

Good catch, BTW.